source/extensions/filters/http/health_check/health_check.cc

```diff
@@ -134,7 +134,7 @@ void HealthCheckFilter::onComplete() {
       for (const auto& item : *cluster_min_healthy_percentages_) {
         details = &RcDetails::get().HealthCheckClusterHealthy;
         const std::string& cluster_name = item.first;
-        const double min_healthy_percentage = item.second;
+        const uint64_t min_healthy_percentage = static_cast<uint64_t>(item.second);
```

Comment:
Does this impact the precision of this? If it does, it might be good to note that in the API docs

Reply:
Yes, it does impact the precision. I've pushed a commit to update the api docs to reflect this.

Should we state in the docs that this will be as of the next release (1.15 presumably)?

Reply:
Also: does it need to be mentioned in the version history?

Reply:
I would add it to the version history under minor behavior changes. That will have the same effect of declaring that this will be included in the next version

Reply:
Great, I've added it.